src/PowerShellEditorServices/Services/DebugAdapter/Handlers/LaunchAndAttachHandler.cs

```diff
@@ -486,6 +522,7 @@ private async Task OnExecutionCompletedAsync(Task executeTask)
             _debugEventHandlerService.UnregisterEventHandlers();
 
             _debugService.IsDebuggingRemoteRunspace = false;
+            _debugService.UnsetPathMappings();
```

Comment:
This is probably not actually something to be solved in this PR, it seems like DebugService needs some sort of finalizer to handle "hey we're wrapping up now." Surprised it's not just disposed and recreated but I'd have to dig in to understand that more.